modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -18,6 +18,7 @@ const mapColors = (colorMap: string[]): Menu.ChoiceMenuItemSpec[] => {
     colors.push({
       text: colorMap[i + 1],
       value: '#' + Transformations.anyToHex(colorMap[i]).value,
+      icon: 'checkmark',
```

Comment:
Is this icon always black? Is the icon visible on a black color swatch?

Reply:
No, thanks to https://github.com/tinymce/tinymce/blob/ea1b5447e8e66e07c1aeba72d978e24b66f723dd/modules/oxide/src/less/theme/components/color-swatch/color-swatch.less#L82 this section this icon is given a border of a different color, so it is visible.

Reply:
That sounds good then! Great work